src/Html2Text.php

```diff
@@ -251,4 +299,20 @@ static function iterateOverNode($node) {
 		return $output;
 	}
 
+	/**
+	 * Accepts a URL (and optionally the link text) and returns a unique index number for that URL.
+	 * @param  string $url  The URL you want an index number for.
+	 * @param  string $text The text of the link (associated with the above URL).
+	 * @return integer      The index number that will refer to the URL passed.
+	 */
+	static function _indexUrl($url,$text=null) {
```

Comment:
How about `private static function indexUrl`? IMHO, underscore in method name is an unhelpful convention in PHP.